drivers/volume/portworx/schedops/schedops.go

```diff
@@ -56,6 +58,31 @@ var (
 	schedOpsRegistry = make(map[string]Driver)
 )
 
+const (
+	defaultDriverNamespace     = "kube-system"
+	defaultPortworxServiceName = "portworx-service"
+)
+
+// GetDriverNamespace returns volume driver namespace
+func GetDriverNamespace() (string, error) {
+	// List all services
+	k8sCore := k8score.Instance()
+	allServices, err := k8sCore.ListServices("", metav1.ListOptions{})
+	if err != nil {
+		return "", fmt.Errorf("cannot list services, Err: %v", err)
+	}
+
+	// Set namespace for driverNamesapce same as portworx-service
+	// if portworx-service is not found, defaultDriverNamespace will be used
+	driverNamespace := defaultDriverNamespace
+	for _, svc := range allServices.Items {
+		if svc.Name == defaultPortworxServiceName {
+			driverNamespace = svc.Namespace
```

Comment:
add _break_ here, as you don't want to continue the loop if you found a namespace